backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDocumentResponseDto.kt

```diff
@@ -0,0 +1,8 @@
+package com.nosql.document.dto
+
+class PersonDocumentResponseDto(
```

Comment:
use `data class`

Reply:
this make no sense, we do not use toString